test/test_configuration.rb

```diff
@@ -12,7 +12,45 @@ def test_change_in_configuration_keep_default_untouched
   end
 
   def test_options
+    hundle_options
     assert @config.options
     assert_includes @config.options, :args
   end
+
+  def test_option_templates
+    hundle_options('--templates')
+    assert(@config.options)
+    assert_includes(@config.options, :templates)
+  end
+
+  def test_option_version
+    hundle_options('-v')
+    assert(@config.options)
+    assert_includes(@config.options, :version)
+  end
+
+  def test_invalid_option
+    assert_throws_and_match(Gem2Rpm::Configuration::ConfigurationError,
```

Comment:
The documentation for the old minitest is not expressive enough. The recent docuementation [1] states:

> Returns the exception matched so you can check the message, attributes, etc.

But this should apply also to old versions of minitest, if I am not mistaken.

[1] http://www.rubydoc.info/gems/minitest/Minitest/Assertions#assert_raises-instance_method